scripts/config.js

```diff
@@ -22,8 +22,8 @@ exports.MUTED_PREFIX = '[white](muted)';
 exports.bannedWords = (function (words) {
     return words.map(function (word) { return (typeof word == "string" || word instanceof RegExp) ? [word, []] : [word[0], word.slice(1)]; });
 })([
-    "uwu",
-    "nig" + "ger", "nig" + "ga", "niger", "ni8" + "8ger",
+    "uwu", "owo", //lol
+    "nig" + "ger", "nig" + "ga", "niger", "ni8" + "8ger", //our apologies to citizens of the Republic of Niger
```

Comment:
`.js` files are compiled from `.ts` files, editing the comments here has no effect (they will be gone next time i change anything else)